### PR TITLE
Bug # 1160364 rounded category icon strokes (+)

### PR DESCRIPTION
**Pull request by rteja91**
Fixed this Css issue of Category links as suggested

### CODE REVIEW

File: src/media/css/elements--categories.styl [hunk 1]
```diff
@@ -63,9 +63,9 @@ mkt-category-item {
         text-decoration: none;
     }
     +media--wide-tablet() {
-        border: 1px solid rgba($greyscale-white, 0.2);
+        border: 2px solid rgba($greyscale-white, 0.2);
         margin: 0;
-
+        border-radius : 5px;
```
Comment (ngokevin) [inline src/media/css/elements--categories.styl]: can you alphabetize the CSS rules and keep the whitespace?

Comment (rteja91) [inline src/media/css/elements--categories.styl]: sorry, i will do that and send it

File: src/media/css/elements--categories.styl [hunk 2]
```diff
@@ -63,9 +63,9 @@ mkt-category-item {
         text-decoration: none;
     }
     +media--wide-tablet() {
-        border: 1px solid rgba($greyscale-white, 0.2);
-        margin: 0;
-
+        	border: 2px solid rgba($greyscale-white, 0.2);        
+        	border-radius : 5px;
+        	margin: 0;
```
Comment (ngokevin) [inline src/media/css/elements--categories.styl]: can you remove the extra whitespace here? We keep our CSS at 4-space indents. Thanks, getting closer!

File: src/media/css/elements--categories.styl [hunk 3]
```diff
@@ -63,9 +63,9 @@ mkt-category-item {
         text-decoration: none;
     }
     +media--wide-tablet() {
-        border: 1px solid rgba($greyscale-white, 0.2);
-        margin: 0;
-
+    border: 2px solid rgba($greyscale-white, 0.2);        
+    border-radius : 5px;
+    margin: 0;
```
Comment (ngokevin) [inline src/media/css/elements--categories.styl]: it should be 4 spaced indents like so:

```
+media--wide-tablet() {
    <RULE1>
    <RULE2>

    &:hover {
        <RULE3>
    }
}
```

**Comment by ngokevin** [comment]
looks great! next time, you can squash your commits into one, but let's get it in anyways.

**Pull request merged**